ChangeLog.md

```diff
@@ -1,4 +1,7 @@
 # Changelog for polysemy-zoo
+## 0.8.0.0
+- Removed IdempotentLowering since “IdempotentLowering is old, unreliable, outclassed by Final, and deprecated.”
```

Comment:
Have this under a `Breaking Changes` sub-sub-section (see other releases in the changelog), and put it under `Unreleased Changes` instead of 0.8.0.0. `Unreleased Changes` is currently at the bottom of this file, but we've learned the hard way in `polysemy` that's bad -- please move it to the top, where you currently have 0.8.0.0.

As for the message, I suggest the following:
```
- Removed `Polysemy.IdempotentLowering`, previously deprecated in favor of `Polysemy.Final`. 
```